### PR TITLE
Companion for new Trie cache

### PR DESCRIPTION
**Pull request by bkchr**
https://github.com/paritytech/substrate/pull/11407

cumulus companion: https://github.com/paritytech/cumulus/pull/1361

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs. 
 The job name - test-linux-stable 
 The job logs - https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1754792

File: runtime/rococo/src/lib.rs [hunk 1]
```diff
@@ -717,20 +717,6 @@ impl pallet_beefy_mmr::Config for Runtime {
 	type BeefyDataProvider = ParasProvider;
 }
 
-parameter_types! {
```
Comment (cheme) [inline runtime/rococo/src/lib.rs:720]: Is it meant to be part of companion?

Comment (bkchr) [reply runtime/rococo/src/lib.rs:720]: Sort of yes. They are not being used anymore. I removed them while fixing some merge conflicts. I can bring them back, however creating an extra pr for this isn't it really worth :P

Comment (cheme) [reply runtime/rococo/src/lib.rs:720]: No it's fine as long as it is not a mistake.

**Review by cheme — APPROVED** [review]

**Review by skunert — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge force

**Pull request merged**